Add-ons/Starter-Kit-TEMP/Widgets/index.md

```diff
@@ -114,10 +114,15 @@ For the best experience with the Slider Widget, we recommend adding multiple ***
 
 ![movePage.jpg](images/The-Tabs-Widget.png)
 
-This allows you to create multiple widgets within one widget this means you could have for example: a ***Text Widget***, an ***Image Widget*** and a ***Forms Widget*** all collected in one.
+The Tabs Widget allows you to create tabs like the ones you find in your browser, these tabs can contain any Widget you desire.
+This allows you to create multiple Widgets within one Widget this means you could have for example: a ***Text Widget***, an ***Image Widget*** and a ***Forms Widget*** all collected in one.
 
+Where the ***Text Widget*** and the ***Image Widget*** is tab 1 and ***Forms Widget** is tab2. 
```

Comment:
```suggestion
Where the ***Text Widget*** and the ***Image Widget*** is tab 1 and ***Forms Widget*** is tab2. 
```

Reply:
I'm a bit confused about this. Does it belong to the section above, where you say it can all be in one, or is this a different setup?

In either case, could you make that more clear some how? 🙂

Reply:
yea what i meant here is the tabs inside the one widget